strapi/strapi
# [Data transfer] Exclude optional admin types in schema check

### What does it do?

Excludes admin types that are non-transferrable and can be disabled (eg, by switching between EE or CE) from the schema check on a data transfer. Currently only audit-logs, but should probably include review workflows before the next version

Note: We are currently planning to add a new strategy that only validates the schema for content types that are included in the transfer, and then making that the default for data transfer. So this is a temporary fix that will ensure audit logs and review workflows work, and architectural fixes around how this is working will come later.

### Why is it needed?

Currently it is impossible to transfer data between a CE and EE strapi even if the schemas are otherwise the same, because the presence of the audit logs causes a schema validation error.

### How to test it?

Use `strapi transfer --to` a site that has audit logs enabled from a site that does not have audit logs, but with otherwise identical schemas. Prior to this PR there will be a schema validation check error for audit logs; with this PR the transfer will work.

### Related issue(s)/PR(s)

Let us know if this is related to any issue/pull request
